google/cloud/storage/client.h

```diff
@@ -1823,6 +1824,36 @@ class Client {
   }
   //@}
 
+  //@{
+  /**
+   * Create a signed URL for the given parameters.
+   *
+   * @param verb the operation allowed through this signed URL, `GET`, `POST`,
+   *     `PUT`, etc. are valid values.
+   * @param bucket_name the name of the bucket.
+   * @param object_name the name of the object, note that the object may not
+   *     exist for signed URLs that upload new objects.
+   * @param options a list of optional parameters for the signed URL, this
+   *     include: `ExpirationTime`, `MD5HashValue`, `ContentType`, and
```

Comment:
Add a note that ExpirationTime defaults to 7 days. Expiration time longer than 7 days using ADC on Compute Engine and GAE Flex will also expire in 7 days due to how Google manages private keys.

https://cloud.google.com/iam/docs/understanding-service-accounts#managing_service_account_keys

Reply:
Done.